_providers/yandex.ru.md

```diff
@@ -25,7 +25,6 @@ last_checked: 2020-03
 website: https://yandex.ru/
 ---
 
-To use Delta Chat with your Yandex accounts you have to set IMAP protocol option *turned on* in the Yandex web interface. Please visit https://yandex.com/support/mail/mail-clients.html for more details.
-
-Russian help: https://yandex.ru/support/mail/mail-clients.html 
+To use Delta Chat with your Yandex accounts you have to set IMAP protocol option *turned on* in the Yandex web interface. Please visit <https://yandex.com/support/mail/mail-clients/others.html> for more details.
 
+Russian help: <https://yandex.ru/support/mail/mail-clients/others.html>
```

Comment:
Then "Russian help" needs to be translated maybe

Reply:
A matter of another MR perhaps. It was this way before me.